bin/rspecq

```diff
@@ -107,9 +113,10 @@ else
   worker = RSpecQ::Worker.new(
     build_id: opts[:build],
     worker_id: opts[:worker],
-    redis_host: opts[:redis_host]
+    redis_host: opts[:redis_host],
   )
 
+  worker.fail_fast = opts[:fail_fast] if opts[:fail_fast]
```

Comment:
If we make the default `false` as I suggested above, this can be simplified to:

```ruby
worker.fail_fast = opts[:fail_fast]
```

Reply:
Also, move this at the end of the list (below `file_split_threshold`) since it semantically belongs there.

Reply:
default is 0

Reply:
Yes sorry, my bad. Anyway, the suggestion still holds.